package.json

```diff
@@ -49,7 +49,19 @@
           "when": "resourceLangId == vsix"
         }
       ]
-    }
+    },
+    "configuration": [
+      {
+        "title": "Install .VSIX",
+        "properties": {
+          "installVSIX.installationForced": {
+            "type": "boolean",
+            "default": false,
+            "description": "Installation forced."
```

Comment:
The description is not descriptive enough, perhaps change it to "Use the --force option, forcing the installation"